modules/ballerina-builtin/src/main/ballerina/ballerina/util/arrays/arrays.bal

```diff
@@ -0,0 +1,22 @@
+package ballerina.util.arrays;
```

Comment:
Why do we need a new file here? Can't we add this to https://github.com/ballerinalang/ballerina/blob/master/modules/ballerina-builtin/src/main/ballerina/ballerina/util/arrays/natives.bal

Reply:
Since this is not a native function, it'll be wrong to add this to `natives.bal`. We have followed the same approach for **caching** as well.